student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(name: 'Unknown', age: nil, parent_permission: true, classroom: 'No Classroom')
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] As I suggested in the previous comments  ,   `age` variable , it should not be defined with a default value as it is not a optional variable. For the student class ,    `classroom` should not be an optional varibale as well, Kindly fix that 😄